### DISCUSSION TITLE
feat: Add Capacitor to Vue-Awesome

### DISCUSSION THREAD

**Discussion by mesqueeb** [ionic-team/capacitor#5319] [Ideas]
### Description

[Vue-Awesome](https://github.com/vuejs/awesome-vue) is a list of awesome Vue related projects that a lot of developers use.

Especially to find projects for compiling Vue apps for mobile development, it's a good place where developers can come look at the options.

Currently the competitors: Weex seems to be prominent in this list, but I was surprised I could not find Capacitor!

I believe it's of high value for marketing purposes that Capacitor is added to this list.

**Comment by jcesarmobile** [comment]
Moving to discussions since this is not a feature for Capacitor.

Anybody can submit anything into that repository, doesn't need to be us, so feel free to add Capacitor to the list.
